src/oas.js

```diff
@@ -128,6 +203,23 @@ class Oas {
     const operation = getPathOperation(this, { swagger: { path }, api: { method } });
     return new Operation(this, path, method, operation);
   }
+
+  findOperation(logInput) {
+    const { url, method } = logInput.request.log.entries[0].request;
```

Comment:
We're gonna move away from referencing `.request.log.entries` (we've "exploded" the request object in the HAR object to the root level of the mongo document)!  Let's use `logInput.url` & `logInput.method`

Reply:
Has that been updated in that repo yet?

Reply:
Good catch, I'll swap that out! Thanks @dok

Reply:
I'd also suggest supplying `findOperation` with the url and method upfront. This package shouldn't need to know about how we store logs in the DB, just that it needs to look for an operation that match a specific pattern.

Reply:
Good call @erunion, i'll move it to the FE.